_posts/randy/2022-10-02-all-creative-now.md

```diff
@@ -0,0 +1,129 @@
+
+Social media outlets like [Snapchat](https://variety.com/2021/digital/news/snapchat-creator-payout-250-million-1235132541), [Pinterest](https://www.engadget.com/pinterest-creator-fund-creator-code-150013929.html) and many [needy Walled-Gardens](https://www.shopify.com/blog/creator-fund) have been throwing million$ in SCRATCH at the CREATIVE ITCH, knowing the next wave of content CREATOR apps may leave them beached and abandoned.
+
+Throwing around pretend money (so they can pretend they are a player) is the high-tech corporate version of socialism – a bribe offered to people with the money made off same peoples' backs. $250 million can't buy you one professional baseball player - maybe we are all playing the wrong game.
+
+CREATIVITY, however difficult - has never been so popular, both from a consumption and a delivery standpoint. While there are many more opportunities than ever before, these new digital markets are overwhelmed with many more micro-competitors.
+
+Flip through the online design for hire site [fiverr](www.fiverr.com) for fiverr minutes be dually amazed: 1) there are so many custom project designers for hire so cheaply, and 2) you have been replaced by the internet and have no chance. Why would anyone pay thousands to you face when they can pay hundreds to someone far away? On the other hand, how does anyone that wants to get rich get there at $5 or $100 and up?
+
+There will be various descriptions of the coming ages, eras or economies since the Information Age is old aged by now. A new BUZZWORD developing is in recognition of the expected CREATIVE ECONOMY. Already 25% of the US workforce toils in what is considered a CREATIVE ROLE.
+
+## TURN ON, TUNE IN – DON'T DROP OUT
+
+Our youth are not shy and will do very well with phones on face and faces on phone. Hopefully they will discover phones can also take some pictures and videos horizontally, where viewers can see fewer skies and bellies, the whole group in the "selfie" shot, and wide format landscapes. We can say it was their invention - we can be dazzled as they rotate +/- 90 degrees.
+
+Rather than be turned away by naysayers, people of all ages should embrace the technologies in our hands and in our faces. Accept your fate, and turn on your devices – get ready to learn. It will take days, months and years to figure out the "RITHYM" and the "RHYME" of your CREATIVE STREAK, but you have a few. Do not worry that you will fall behind, as that already happened again while his sentence was being  composed.
+
+The best players at any game are the most talented players that never give up. If all odds are against the SELF-STARTER, the payoff for a win is going to be pretty big. The attraction to be a CREATOR, an INFLUENCER, a CREATOR, may secretly be a FORCE within a lot of us.
+
+...
+
+Randy Berg
+xyz digital inc.
+
+...
+
+## NAB AMPLIFY – AMPLIFICATION
```

Comment:
Does this intentionally come after signature?

Reply:
Maybe I'm just tired, but this feels out of place

Reply:
was intentional - block from one source